streams/src/main/java/org/apache/kafka/streams/processor/To.java

```diff
@@ -65,4 +67,22 @@ public To withTimestamp(final long timestamp) {
         this.timestamp = timestamp;
         return this;
     }
+
+    @Override
+    public boolean equals(final Object o) {
+        if (this == o) {
```

Comment:
Could we use `if/else if/else` for this part of the logic? It could be more reader-friendly.

Reply:
Not sure what you mean? Btw this code is generated by IntelliJ